docs/versions.json

```diff
@@ -1,12 +1,20 @@
-[
-  {
-    "version": "4.1.3",
-    "name": "4.1.3",
-    "path": "/nebular"
-  },
-  {
-    "version": "3.6.2",
-    "name": "3.6.2",
-    "path": "/nebular/3.6.2"
-  }
-]
+{
+  "currentVersionName": "4.1.3",
```

Comment:
should we simply add `current`: true to the versions declaration?

Reply:
done, thanks